manage-data/data-store/data-streams/set-up-tsds.md

```diff
@@ -11,103 +11,38 @@ products:
+        - `manage_index_templates` for creating a template to base the TSDS on
+    - [Index privileges](elasticsearch://reference/elasticsearch/security-privileges.md#privileges-list-indices) 
+        - `create_doc` and `create_index` for creating or converting a TSDS
+        - `manage` to [roll over](#convert-existing-data-stream-to-tsds) a TSDS
 
-## Prerequisites [tsds-prereqs]
+## Set up a TSDS
 
-* Before you create a TSDS, you should be familiar with [data streams](../data-streams.md) and [TSDS concepts](time-series-data-stream-tsds.md).
-* To follow this tutorial, you must have the following permissions:
+:::::{stepper}
+::::{step} Create an index template 
+:anchor: create-tsds-index-template
 
-    * [Cluster privileges](elasticsearch://reference/elasticsearch/security-privileges.md#privileges-list-cluster): `manage_ilm` and `manage_index_templates`.
-    * [Index privileges](elasticsearch://reference/elasticsearch/security-privileges.md#privileges-list-indices): `create_doc` and `create_index` for any TSDS you create or convert. To roll over a TSDS, you must have the `manage` privilege.
+The structure of a time series data stream is defined by an index template. Create an index template with the following required elements and settings:
 
+- **Index patterns:** One or more wildcard patterns matching the name of your TSDS, such as `weather-sernsors-*`. For best results, use the [data stream naming scheme](/reference/fleet/data-streams.md#data-streams-naming-scheme).
+- **Data stream object:** The template must include `"data_stream": {}`.
+- **Time series mode:** Set `index.mode: time_series`.
+- **Field mappings:** Define at least one dimension field and typically one or more metric fields:
+    - To define a dimension, set `time_series_dimension` to `true`. For more details, refer to [Dimensions](/manage-data/data-store/data-streams/time-series-data-stream-tsds.md#time-series-dimension). 
+        - To define dimensions dynamically, you can use a pass-through object. For details, refer to [Defining sub-fields as time series dimensions](elasticsearch://reference/elasticsearch/mapping-reference/passthrough.md#passthrough-dimensions).
+    - To define a metric, use the `time_series_metric` mapping parameter. For more details, refer to [Metrics](/manage-data/data-store/data-streams/time-series-data-stream-tsds.md#time-series-metric).
+    - (Optional) Define a `date` or `date_nanos` mapping for the `@timestamp` field. If you don't specify a mapping, {{es}} maps `@timestamp` as a `date` field with default options.
+    * (Optional) Other index settings, such as [`index.number_of_replicas`](elasticsearch://reference/elasticsearch/index-settings/index-modules.md#dynamic-index-number-of-replicas), for the data stream's backing indices.
+- A priority higher than `200`, to avoid [collisions](/manage-data/data-store/templates.md#avoid-index-pattern-collisions) with built-in templates.
```

Comment:
I think we need to add here a lifecycle management section. I would recommend adding the following in the `template`:
```
"lifecycle": {
  "enabled": true
}
```
The main reason we need this is rollover, if a user doesn't add this they are going to end up with a gigantic index. Everything else is optional, we can suggest that users can read further on lifecycle management if they want more features like retention or tiers.